recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/RecipientsResolver.java

```diff
@@ -1,93 +1,89 @@
 package com.redhat.cloud.notifications.recipients.resolver;
 
-
 import com.redhat.cloud.notifications.recipients.model.RecipientSettings;
 import com.redhat.cloud.notifications.recipients.model.User;
 import io.quarkus.cache.CacheResult;
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.inject.Inject;
 import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-import java.util.stream.Collectors;
-
 
+import static java.util.stream.Collectors.toSet;
 
 @ApplicationScoped
 public class RecipientsResolver {
```

Comment:
Review tip: don't compare the old and new codes. Reading the new code and comments should be clear enough.